backend/Domain/Models/Company.cs

```diff
@@ -12,6 +12,5 @@ public class Company : IdentityUser
     {
         public ICollection<CompanyService>? CompanyServices { get; }
 
-        public string? FullName { get; set; }
```

Comment:
Please re add this, since I'm working in the company entity, and I will remove this part, but yeah, good that you removed all the others.